### PR TITLE
[MM-49544] - Show which SKU is being removed when removing license

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
Show which SKU we're removing when downgrading server

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-49544

```release-note
NONE
```

### CODE REVIEW

**Review by yasserfaraazkhan — APPROVED** [review]
Hi Allan, 
we can see the SKU name being update in the modal,

Screenshot 2023-01-11 at 8 17 17 PM

Screenshot 2023-01-11 at 8 17 46 PM


Screenshot 2023-01-11 at 8 17 04 PM

**Review by Afsoon — APPROVED** [review]
LGTM @AGMETEOR!

File: utils/license_utils.ts [hunk 1]
```diff
@@ -94,3 +94,8 @@ export const isEnterpriseLicense = (license?: ClientLicense) => {
 };
 
 export const isNonEnterpriseLicense = (license?: ClientLicense) => !isEnterpriseLicense(license);
+
+export const licenseSKUWithFirstLetterCapitalized = (license: ClientLicense) => {
```
Comment (marianunez) [inline utils/license_utils.ts:98]: naming nit : 

```suggestion
export const licenseSKUTitleCase = (license: ClientLicense) => {
```

File: components/admin_console/license_settings/modals/confirm_license_removal_modal.test.tsx [hunk 2]
```diff
@@ -89,4 +90,14 @@ describe('components/admin_console/license_settings/modals/confirm_license_remov
...
+
+    test('should show which SKU is currently being removed in confirmation message', () => {
+        const wrapper = mountWithIntl(
+            <Provider store={store}>
+                <ConfirmLicenseRemovalModal {...props}/>
+            </Provider>,
+        );
+
+        expect(wrapper.find('ConfirmLicenseRemovalModal').find('div.subtitle').find('span').text()).toEqual('Removing the license will downgrade your server from Professional to Free. You may lose information. ');
+    });
```
Comment (marianunez) [inline components/admin_console/license_settings/modals/confirm_license_removal_modal.test.tsx:102]: Nice 👍

**Review by marianunez — APPROVED** [review]
Thanks @AGMETEOR!

**Pull request merged**